supabase/migrations/20241030103924_add-first_message_id-on-topics.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE topics
+    ADD COLUMN first_message_id text REFERENCES messages(id) ON DELETE SET NULL;
+
```

Comment:
You might as well add an index on the `first_message_id` column while you're at it to ensure faster lookups? 🤞